vector/src/main/java/im/vector/app/core/date/VectorDateFormatter.kt

```diff
@@ -19,64 +19,147 @@ package im.vector.app.core.date
 import android.content.Context
 import android.text.format.DateFormat
 import android.text.format.DateUtils
+import im.vector.app.core.resources.DateProvider
 import im.vector.app.core.resources.LocaleProvider
+import im.vector.app.core.resources.toTimestamp
 import org.threeten.bp.LocalDateTime
+import org.threeten.bp.Period
 import org.threeten.bp.format.DateTimeFormatter
-import java.util.Calendar
-import java.util.Date
 import javax.inject.Inject
-
-/**
- * Returns the timestamp for the start of the day of the provided time.
- * For example, for the time "Jul 21, 11:11" the start of the day: "Jul 21, 00:00" is returned.
- */
-fun startOfDay(time: Long): Long {
```

Comment:
This was added by a PR from the community: https://github.com/vector-im/element-android/pull/1803 to fix #822 

Is it OK to remove it?

Reply:
Yes, I don't use the relative method anymore